package.json

```diff
@@ -1763,6 +1763,18 @@
 							"onCommand:openshift.Serverless.createFunction"
 						]
 					},
+					{
+						"id": "createFunctionWithGitTemplate",
+						"title": "Create Function using git template",
+						"description": "Creates a Function in the mentioned directory with the specified git based template selected and handles HTTP events. \n[Create Function](command:openshift.Serverless.createFunction)",
```

Comment:
What does "and handles HTTP events" mean here? It's not clear to me what this is saying. I think you can remove it.